can_read_write.py

```diff
@@ -0,0 +1,94 @@
+# This scipt must run on Pi startup
+
+import can
+import time
+import os
+import sys
+import json
+from threading import Thread
+
+# info handled; specified by https://docs.google.com/document/d/1kUU54jQZAB9nwCM-iA96Kj0fXJ6RNw9nAcBffzji5cU/edit
+# full list of PIDs at https://en.wikipedia.org/wiki/OBD-II_PIDs
+PIDS = {'VEHICLE_CHARGE': 0x2F,
```

Comment:
Could you either change the name of this array or add a comment clarifying that PID means Parameter ID? When I see PID I think of "Process ID."